packages/front-end/components/Settings/InviteModal.tsx

```diff
@@ -10,6 +10,12 @@ import InviteModalSubscriptionInfo from "./InviteModalSubscriptionInfo";
 import useStripeSubscription from "../../hooks/useStripeSubscription";
 import UpgradeModal from "./UpgradeModal";
 
+type inviteResult = {
```

Comment:
Types should start with capital letters.

Also, we should either remove `emailSent` from this type or use a single state array instead of separate success/failure arrays.

Reply:
I opted to drop `emailSent` from the `InviteResult` as opposed to using a single state array as I couldn't come up with a good solution for rendering the results as I have them now via a single state array.